crates/dekoder/src/inventory.rs

```diff
@@ -0,0 +1,87 @@
+//! Assets manager.
+use glob::glob;
+use hashbrown::HashMap;
```

Comment:
Why are you using `hashbrown::HashMap` instead of `std::collections::HashMap`? I believe the latter should even link to the former, so to keep the number of depenencies down you should use the standard modules.

Reply:
ehm @alecandido suggested me to prefer the first over the second, but I can also change - I've no idea :see_no_evil: ... done in d888c339; this should then also be reflected on the `ekore` side ...

Reply:
Well, there were many reasons, mainly listed on the repo README itself: https://github.com/rust-lang/hashbrown (memory, performances, compatibility with `no_std`).

It is maintained by the https://github.com/rust-lang/ org itself, and it is compatible with PyO3 (since a long time).
https://pyo3.rs/v0.22.2/conversions/tables

Reply:
please tell me if I should revert that commit or not :innocent: (in wise prediction I already put it in a single commit :see_no_evil: )

Reply:
No, the README confirm what I said above:

> Since Rust 1.36, this is now the HashMap implementation for the Rust standard library.

Reply:
Don't get me wrong: it's also fine to use the `std` one, to have one dependency less. I was just explaining that there might be motivations to take that into account (most likely not that relevant here, since you have few big elements, rather than many small ones).